src/hotspot/os/linux/cgroupSubsystem_linux.hpp

```diff
@@ -72,31 +72,37 @@
-    log_trace(os, container)(log_string " failed: %d", OSCONTAINER_ERROR);            \
-    return OSCONTAINER_ERROR;                                                         \
+    log_trace(os, container)(log_string " failed: -2");                               \
+    return false;                                                                     \
   }                                                                                   \
-  log_trace(os, container)(log_string " is: " JULONG_FORMAT, retval);                 \
+  log_trace(os, container)(log_string " is: " UINT64_FORMAT, retval);                 \
+  return true;                                                                        \
 }
 
 #define CONTAINER_READ_NUMBER_CHECKED_MAX(controller, filename, log_string, retval)   \
 {                                                                                     \
   bool is_ok;                                                                         \
-  is_ok = controller->read_number_handle_max(filename, &retval);                      \
+  is_ok = controller->read_number_handle_max(filename, retval);                       \
   if (!is_ok) {                                                                       \
-    log_trace(os, container)(log_string " failed: %d", OSCONTAINER_ERROR);            \
-    return OSCONTAINER_ERROR;                                                         \
+    log_trace(os, container)(log_string " failed: -2");                               \
+    return false;                                                                     \
   }                                                                                   \
-  log_trace(os, container)(log_string " is: " JLONG_FORMAT, retval);                  \
+  if (retval == value_unlimited) {                                                    \
+    log_trace(os, container)(log_string " is: -1");                                   \
+  } else {                                                                            \
+    log_trace(os, container)(log_string " is: " UINT64_FORMAT, retval);               \
+  }                                                                                   \
+  return true;                                                                        \
 }
 
 #define CONTAINER_READ_STRING_CHECKED(controller, filename, log_string, retval, buf_size) \
 {                                                                                         \
   bool is_ok;                                                                             \
   is_ok = controller->read_string(filename, retval, buf_size);                            \
   if (!is_ok) {                                                                           \
-    log_trace(os, container)(log_string " failed: %d", OSCONTAINER_ERROR);                \
+    log_trace(os, container)(log_string " failed: -2");                                   \
```

Comment:
Why this change? Did the constant value change?

Reply:
Motivation was getting rid of the OSCONTAINER_ERROR constant. The only place where a negative number was still in use. I've just dropped the `: -2` suffix now. It's not very useful (other than in tests).